src/lib/receptions/MockReception.js

```diff
@@ -25,6 +25,30 @@ const buildMockSampleAndRequest = (sample_name, requestOptions) => {
   }
 }
 
+/**
+ * A function to help encapsulate the logic of building a mock sample and request object.
+ *
+ * @param {string} sample_name - The name of the sample to be included in the mock data.
+ * @param {Object} requestOptions - An object containing request options to be included in each well's request.
+ * @returns {Object} A mock sample and request object.
+ */
+const buildMockCompoundSampleAndRequest = (sample_names, requestOptions) => {
```

Comment:
Since this mostly a copy of `buildMockSampleAndRequest` but as a list, could they be combined or could this be removed?

Reply:
Good point, It is good to refactor `buildMockCompoundSampleAndRequest`  to use `buildMockSampleAndRequest` to create that list, so there wont be duplication.